index.yaml

```diff
@@ -6378,36 +6378,6 @@ entries:
     - assets/rancher-alerting-drivers/rancher-alerting-drivers-1.0.100.tgz
     version: 1.0.100
   rancher-backup:
-  - annotations:
```

Comment:
@vardhaman22 why is this PR removing this `rancher-backup` entry on `index.yaml`?

Reply:
updated..theses changes are no longer present ..fixed by https://github.com/rancher/charts/pull/3613